README.md

```diff
@@ -231,7 +231,15 @@ plugins:
     author: "Ansible Cloud Team"
 ```
 
```

Comment:
When you list `generate_examples` in the section  **Input Parameters:** (line 110), would it make sense for you to add a Note like `generate_examples` is not supported for vmware and link this section to see how to do it?

Reply:
@alinabuzachis generate_examples is supported by vmware. Please let me know if I am missing something here.